### ISSUE TITLE
YPT Live streamer not showing youtube links

### ISSUE THREAD

**Issue by milehighkaraoke** [WWBN/AVideo#8333]
@DanielnetoDotCom 
links ard showing on web page and mobile page but not the YPT Live app
![Screenshot_20230815_204431_Firefox](https://github.com/WWBN/AVideo/assets/70327833/9ca49319-5060-4b9e-9d7b-8f9f74b6e551)
![Screenshot_20230815_204313](https://github.com/WWBN/AVideo/assets/70327833/fde5e389-2f8f-46ff-aed2-90e8924021d1)

**Comment by DanielnetoDotCom** [comment]
thanks I will review it in the next update

**Comment by milehighkaraoke** [comment]
You can see the one above in the two screenshots about Joe Biden is missing, that is a YouTube link it's available on computer and the mobile but it's not available on the ypt application

**Comment by DanielnetoDotCom** [comment]
is this an embed link?

the YPT app does not support embed yet

**Comment by milehighkaraoke** [comment]
yes I do beleive it is. Another user shared it like that and I noticed it was not coming up in the ap. thanks for clarification
